cmd/signozschemamigrator/schema_migrator/metrics_migrations.go

```diff
@@ -362,7 +362,6 @@ var MetricsMigrations = []SchemaMigrationRecord{
 					{Name: "last_reported_unix_milli", Type: SimpleAggregateFunction{FunctionName: "max", Arguments: []ColumnType{ColumnTypeUInt64}}, Codec: "ZSTD(1)"},
 				},
 				Engine: Distributed{
-					Cluster:     "cluster",
```

Comment:
This will be covered by `ON CLUSTER` https://github.com/SigNoz/signoz-otel-collector/blob/bd0b408812efb30ad8f350550ceaf74f08f7252c/cmd/signozschemamigrator/schema_migrator/table.go#L191-L194